GeneralKnowledge/internet/internetConcept/readme.md

```diff
@@ -0,0 +1,124 @@
+# Internet
+
+#### Some terminology to look out for....
+
+- Network (Devices that are connected to each other....) whereas Internet (global network of networks communicate through a standardized set of protocols)
+- Internet is just the network of the networks.
+- Decentralization - Nobody owns/controls it.
+
+---
+
+## Domain Name System(DNS)
+ - A domain name gets the IP address rather than you need to memorize address such as 111.111.111.111
+ - First, when url is clicked. It will check cache(which can be OS/browser/router) otherwise on internet it will check the 4 servers.
+ - Recursive server - DNS resolver which keeps on asking start from root to TLD to Authoritative. Afterwards, it'll cache.
```

Comment:
keeps on enquiring....starting from root.............it'll cache on the DNS cache